contrib/ext_editor.lua

```diff
@@ -334,30 +332,50 @@ local function export2collection(storage, image_table, extra_data)
     new_image:group_with(image.group_leader)
     end 
   
-  dt.print (_("finished exporting"))
+  dt.print(_("finished exporting"))
   end
 
--- install the module in the UI
-local function install_module()
+
+-- install the module in the UI -----------------------------------------------
+local function install_module(dr)
   if not ee.module_installed then
-    -- register new module "external editors" in lighttable ------------------------
-    dt.register_lib(
-      MODULE_NAME,          
-      _("external editors"),  
-      true, -- expandable
-      false,  -- resetable
-      {[dt.gui.views.lighttable] = {"DT_UI_CONTAINER_PANEL_RIGHT_CENTER", 100}},  
-      dt.new_widget("box") {
-        orientation = "vertical",
-        table.unpack(ee.widgets),
-        },
-      nil,  -- view_enter
-      nil   -- view_leave
-      )
+    if dr then
+      -- register new module "external editors" in lighttable and darkroom ----
```

Comment:
you could do 

```
local views = {[dt.gui.views.lighttable] = {"DT_UI_CONTAINER_PANEL_RIGHT_CENTER", 100}}
if dr then 
views = {[dt.gui.views.lighttable] = {"DT_UI_CONTAINER_PANEL_RIGHT_CENTER", 100},
         [dt.gui.views.darkroom] = {"DT_UI_CONTAINER_PANEL_LEFT_CENTER", 100}}
end
```

then just replace the view part with the variable views and so only have the one code block for installing the module

Reply:
implemented